update console readme (#946)
hasura/graphql-engine: console/README.md

# Hasura Console

> The Hasura console is an admin dashboard used to manage the connected database and to experiment with GraphQL APIs. It is a React application bundled with webpack and the state is managed using Redux.

Reviewer: This change isn't required and actually changes the intended meaning.

### Development with Hasura GraphQL Engine (`server` mode)

Hasura GraphQL engine should be running to develop console in this mode. If you have set it up on Heroku, your url will look like `<app-name>.herokuapp.com`, if it's on your local machine, it's probably `http://localhost:8080`.

> [Dotenv](https://github.com/motdotla/dotenv) is used for setting environment variables for development. Create a `.env` file in the root directory for console (wherever package.json is). Sample `.env` file can contain the example environment variables below:

Reviewer: ```suggestion
[Dotenv](https://github.com/motdotla/dotenv) is used for setting environment variables for development. Create a `.env` file in the root directory for console (wherever package.json is). Here's a `.env` file with some environment variable examples :
```
